bcgov/mds
# MDS-3846 - separate docman from build.yaml into separate github action

# Main

- What did you focus on in this PR?

Creating a separate GitHub action for document-manager.

# Other

- What other side fixes, tweaks did you add on the side?

document-manager has been removed from the monolithic build.yaml file and is a separate job that can be viewed as an individual GitHub action. 

# How to test

Trigger GitHub action "Build docman" and ensure it succeeds.

NOTE: As of the time of this PR, lint manifests may fail even if the build is successful. Linting for these builds are a W.I.P and do not work correctly. Thus, these can be ignored for the time being according to @jon-funk .  

# Notes

- Any notes/screenshots for this PR?
- N/A
